srs_data/tagging.py

```diff
@@ -28,7 +32,16 @@ def input_file(self):
             f'{CLIENT_FULL_NAMES[self._client]}-tagsource{self._start_date}.csv'
         )
 
+    @property
+    def uniform_sample_file(self):
```

Comment:
is there a reason this has so many properties? I don't think that any of these things need to be exposed

Reply:
I sort've understand properties as like a lean and name-spaced way to programmatically define attributes in a class instance. Of course, I do the above so I can eventually write `'local://' + self.uniform_sample_file`, etc.